e2e/connect/connect_test.go

```diff
@@ -120,6 +121,30 @@ func testConnectMultiService(t *testing.T) {
 	assertServiceOk(t, cc, "echo2-sidecar-proxy")
 }
 
+// testConnectTransparentProxy tests the Connect Transparent Proxy integration
+func testConnectTransparentProxy(t *testing.T) {
+	_, cleanup := jobs3.Submit(t, "./input/tproxy.nomad.hcl", jobs3.Timeout(time.Second*60))
+	t.Cleanup(cleanup)
```

Comment:
lil tidbit: `jobs3`s cleanup automatically, unless instructed not to (d34788896f8892377a9039b81a65abd7a913b3cc)